### ISSUE TITLE
Can you add eCDP to the Rom List?

### ISSUE THREAD

**Issue by ImPurpl3** [Arisotura/melonDS#885]
The eCrew Development Program can't make saves for manager accounts unless its in the rom list. Thank you!

**Comment by SlyCooperReloadCoded** [comment]
Oh good, someone else opened an issue about this, I was going to.  The name of the rom is "DSCDP_CRUJN6_00.nds" if that matters.

**Comment by RSDuck** [comment]
we already had a request for this in our forums. What we need is the game code and size of the rom and the type of save file, then we can add it to the database.

**Comment by SlyCooperReloadCoded** [comment]
That would be me.  I never got a clear response on how to give you that information without sending you the rom.  Although, there is an archive.org link for it, here it is:
https://archive.org/details/mcdonalds-japan-ecdp-rom-training-nintendo-ds-cartridge-dump

That contains the rom as well as the save file.

**Comment by RSDuck** [comment]
The game code seems to be NTR-CRUJ-JPN, the rom size is 256 MB and the savefile 512 KB. We'll add it soon.

**Comment by RSDuck** [comment]
I didn't test it, but this should work. Please note that melonDS won't try to correct already created savefiles, so you need to delete previously created savefiles.

**Comment by SlyCooperReloadCoded** [comment]
Sorry if I sound stupid, but where's the new version?  Actually, where's any dev/canary builds at all?  There should be release and dev branches, isn't that how emulators work?

**Comment by SlyCooperReloadCoded** [comment]
Okay, I found the new build.  Had to go to the change's Artifacts page, but I found it.  Unfortunately, it still fails to create the save file, claiming that fails to write data.  Can this be re-opened?

**Comment by Arisotura** [comment]
have you tried deleting any existing savefile?

I can look into it quickly

**Comment by SlyCooperReloadCoded** [comment]
Yes, I've deleted all existing save files.  I think it has something to do with the unique structure of the save, it's not like any other gamesave.  Also, where is the direct download link to the latest development build, and why isn't it next to the release build's download link?

**Comment by Arisotura** [comment]
er, I gave it a quick look, and... it's all in Japanese, I can't understand any of it

**Comment by SlyCooperReloadCoded** [comment]
Well, I would provide the english-patched version, but the DSi menu throws an error when booting patched roms, I need an RSA-mismatch-compatible BIOS or something.  The error messages are translated, most of them anyways.

**Comment by Arisotura** [comment]
or run it in DS mode

**Comment by SlyCooperReloadCoded** [comment]
It fails to detect the MAC address if run in DS mode.  This rom truly is a strange one.

**Comment by WaluigiWare64** [comment]
> Well, I would provide the english-patched version, but the DSi menu throws an error when booting patched roms, I need an RSA-mismatch-compatible BIOS or something.  The error messages are translated, most of them anyways.

Have you tried launching it through Unlaunch or TWiLight Menu++?
